docs/quickstart.md

```diff
@@ -103,54 +103,41 @@ This is so we can and provide a seamless user experience for our provided DNS.
 Let's get things going:
 
 ```bash
-./opstrace create aws $OPSTRACE_CLUSTER_NAME \
+./opstrace create aws $OPSTRACE_NAME \
   -c opstrace-config.yaml
 ```
 
-**Be patient:** Cluster creation takes on average 30 minutes on AWS (but it can go as long as 45 minutes).
+**Be patient:** Installation takes on average 30 minutes on AWS (but it can go as long as 45 minutes).
```

Comment:
The installation?

Reply:
I think this is ok, matches "creation" in the original.  Can discuss more later if you want.